doc/usage/restructuredtext/field-lists.rst

```diff
@@ -9,19 +9,60 @@ fields marked up like this::
 
     :fieldname: Field content
 
-Sphinx provides custom behavior for bibliographic fields compared to docutils.
+Field lists are :duref:`originally defined in docutils <field-lists>`
```

Comment:
This section describes about "Bibliographic fields" in Sphinx. So it should refer this section on reST spec. 
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bibliographic-fields

Reply:
is there a way for me to do this with rST? I'm not sure what is the right syntax...

Reply:
I think you want

    :duref:`ref <bibliographic-fields>`